toss_4_x86_64_ib_cray/packages.yaml

```diff
@@ -0,0 +1,86 @@
+packages:
+  all:
+    # This defaults us to machine specific flags of ivybridge which allows
+    # us to run on broadwell as well
+    target: [zen2]
+    compiler: [gcc, cce, clang]
+  cmake:
+    version: [3.14.5]
+    buildable: false
+
+    externals:
+    - spec: cmake
+      prefix: /usr/tce/packages/cmake/cmake-3.14.5
+  cuda:
+    version: [11.4.120]
+    buildable: false
+
+    externals:
+    - spec: cuda@11.4.120
+      prefix: /opt/toss/cudatoolkit/11.4/ 
+  hip:
+    version: [4.2.0, 4.3.1, 4.5.0, 4.5.2]
```

Comment:
Do I need all these versions?

Reply:
I would only include the available versions - if they are all on the machine it's fine to include them.

Reply:
They are all available in the /opt/rocm-4.*/ dirs, so I'll leave it for now!